compiler/testData/codegen/box/functions/localFunctions/kt3978.kt

```diff
@@ -1,11 +1,20 @@
-// IGNORE_BACKEND: JVM_IR
-fun box() : String {
-
+class C() {
```

Comment:
Please keep old test and add new one

Reply:
Done. I split it up but also cleaned up the old test a bit.